src/emailservice/Dockerfile

```diff
@@ -12,14 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM python:3.12.6-slim@sha256:ad48727987b259854d52241fac3bc633574364867b8e20aec305e6e7f4028b26 AS base
+FROM python:3.13.0-alpine@sha256:6fe2f8718e257d8be95f79e3ad3659183f01b0b28fe1d50f5f6e3c54ebbd8f12 AS base
 
 FROM base AS builder
 
-RUN apt-get -qq update \
-    && apt-get install -y --no-install-recommends \
-        wget g++ \
-    && rm -rf /var/lib/apt/lists/*
+RUN apk update \
+    && apk add --no-cache wget g++ \
+    && rm -rf /var/cache/apk/*
```

Comment:
* `alpine` uses `apk`, not `apt-get`.
* The `rm -rf /var/cache/apk/*` line removes package lists and information that apt-get downloads during updates. (It might be redundant for our Dockerfile because we're using a multi-stage build, but I'll leave it for now.)